### ISSUE TITLE
exe version crashes

### ISSUE THREAD

**Issue by mannaggacardarell** [Avnsx/opgg#2]
When i try to launch it, it crashes.
![Screenshot_136](https://user-images.githubusercontent.com/74499805/168427679-b93910d4-a105-4241-a7be-4a252a5ebc97.png)

**Comment by Avnsx** [comment]
If you are connecting to the internet through a proxy or a VPN this might cause the issue

**Comment by mannaggacardarell** [comment]
ok i think the problem was kaspersky. 
Btw now i'm trying to get infos. I add %20 for the space but when i try to look into my profile now i have this problem 

[2] ERROR

Traceback (most recent call last):
  File "opgg_scraper.py", line 34, in <module>
    opgg_update = sess.post('https://'+Region+'.op.gg/summoner/ajax/renew.json/', headers=headers, data={'summonerId': re.findall(r'Id=[0-9]+', str(opgg_website.content))[0].strip('Id=')})
IndexError: list index out of range

**Comment by Avnsx** [comment]
> ok i think the problem was kaspersky. Btw now i'm trying to get infos. I add %20 for the space but when i try to look into my profile now i have this problem
> 
> [2] ERROR
> 
> Traceback (most recent call last): File "opgg_scraper.py", line 34, in opgg_update = sess.post('https://'+Region+'.op.gg/summoner/ajax/renew.json/', headers=headers, data={'summonerId': re.findall(r'Id=[0-9]+', str(opgg_website.content))[0].strip('Id=')}) IndexError: list index out of range

Just enter your normal username, the code already parses it in the needed format for you, without you seeing it.

**Comment by mannaggacardarell** [comment]
I get always the same error

**Comment by Avnsx** [comment]
> I get always the same error

Yeah you're right, seems like op.gg did some updates in the recent months. I'll fix it eventually
